minio-client/deployment/src/main/java/io/quarkiverse/minio/client/deployment/MinioClientProcessor.java

```diff
@@ -2,18 +2,13 @@
 
 import java.util.Map;
 import java.util.function.BooleanSupplier;
+import java.util.function.Supplier;
 
 import jakarta.inject.Singleton;
 
+import io.minio.MinioAsyncClient;
 import io.minio.MinioClient;
-import io.quarkiverse.minio.client.EmptyHttpClientProducer;
-import io.quarkiverse.minio.client.MinioBuildTimeConfiguration;
-import io.quarkiverse.minio.client.MinioClients;
-import io.quarkiverse.minio.client.MinioQualifier;
-import io.quarkiverse.minio.client.MinioRecorder;
-import io.quarkiverse.minio.client.MiniosBuildTimeConfiguration;
-import io.quarkiverse.minio.client.MiniosRuntimeConfiguration;
-import io.quarkiverse.minio.client.WithMetricsHttpClientProducer;
+import io.quarkiverse.minio.client.*;
```

Comment:
One last thing, we use quarkus forma ting guide, and that means no stars for imports, could you please correct it and squash your commits and I will merge.